adapter-commons/pom.xml

```diff
@@ -41,9 +41,28 @@
     </dependencies>
 
     <properties>
-        <java.version>1.8</java.version>
+        <!-- [PIPELINE] -->
+        <distribution.management.release.id>artifactory-local</distribution.management.release.id>
```

Comment:
localhost:8081? should be depend on hardcoded URL?

Reply:
You can override it and it happens via the pipeline